<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>burnConfigDialog</class>
 <widget class="QDialog" name="burnConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVRDUDE Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="eepe.qrc">
    <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLineEdit" name="avrdude_location">
       <property name="toolTip">
        <string>Location of AVRDUDE executable</string>
       </property>
       <property name="whatsThis">
        <string>The location of the AVRDUDE.EXE executable.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="whatsThis">
        <string>Use this button to browse and look for the AVRDUDE.EXE executable file.</string>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Programmer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="avrdude_programmer">
       <property name="whatsThis">
        <string>Programmer used for communicating with the controller.
Please consult the programmer's documentation and the AVRDUDE documentation to select the appropriate programmer.</string>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
       <item>
        <property name="text">
         <string notr="true">c2n232i</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dasa3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dasa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">siprog</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">ponyser</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">89isp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">frank-stk200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">blaster</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">ere-isp-avr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">atisp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dapa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">xil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">futurlec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">abcmini</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">picoweb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">sp12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">alf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">bascom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dt006</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">pony-stk200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">bsd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">pavr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_pdi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_dw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_hvsp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_pp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_isp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">dragon_jtag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2pdi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2avr32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtagmkII_avr32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2dw</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2isp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2fast</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag2slow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtagmkII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag1slow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtag1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">jtagmkI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avr911</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avr109</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">butterfly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">usbtiny</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">usbasp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avr910</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk600hvsp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk600pp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk500hvsp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk500pp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk500v2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">mib510</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk500v1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">stk500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">buspirate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avrisp2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avrispmkII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avrispv2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">avrisp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">arduino</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="whatsThis">
        <string>List all available programmers.</string>
       </property>
       <property name="text">
        <string>List Available</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="avrArgs">
       <property name="toolTip">
        <string>Extra arguments that will be passed to AVRDUDE on every call</string>
       </property>
       <property name="whatsThis">
        <string>Extra arguments used in AVRDUDE.
This can be used for providing extra information to AVRDUDE.

Please only use this if you know what you are doing.  There are no error checks and you could cripple your controller.</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Extra Arguments</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_4">
       <property name="whatsThis">
        <string>Show AVRDUDE help</string>
       </property>
       <property name="text">
        <string>Show Help</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="avrdude_port">
       <property name="whatsThis">
        <string>Communication port to the programmer.
</string>
       </property>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">usb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">com1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">com2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">com3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">com4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">lpt1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">lpt2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">lpt3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">lpt4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>avrdoper</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>AVRDUDE Location</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>mcu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="avrdude_mcu">
       <property name="whatsThis">
        <string>Programmer used for communicating with the controller.
Please consult the programmer's documentation and the AVRDUDE documentation to select the appropriate programmer.</string>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
       <item>
        <property name="text">
         <string>m64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m328p</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="eepe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>burnConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>burnConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
